src/manifest/MsQuicEtw.man

```diff
@@ -3356,6 +3362,15 @@
               template="tid_ID"
               value="11269"
               />
+          <event
+              keywords="ut:Connection ut:DataFlow"
+              level="win:Verbose"
+              message="$(string.Etw.ConnCantGetNewDestCid)"
+              opcode="Connection"
+              symbol="QuicConnCantGetNewDestCid"
+              template="tid_CONN_CANT_GET_NEW_DEST_CID"
+              value="11270"
+              />
```

Comment:
Please but this with (at the end of) the other connection events... But actually, I'm not even 100% sure this extra log is necessary in general.

Reply:
I deleted that log